.annotations_sample

```diff
@@ -1,7 +1,11 @@
-source_path: ../
+source_path: ../devstack-docker/edx-platform/
```

Comment:
was this change intentional, or temporary for testing?

Reply:
Temp, will fix.